scripts/lib.sh

```diff
@@ -28,48 +26,10 @@ export XDG_CONFIG_DIRS="${XDG_CONFIG_DIRS:-/etc/xdg}"
 export XDG_CACHE_HOME="${XDG_CACHE_HOME:-$HOME/.cache}"
 export XDG_RUNTIME_DIR="${XDG_RUNTIME_DIR:-/var/run}"
 
-
-# Check if a command is available
 require() {
     command -v "$1" >/dev/null 2>&1
 }
 
-query-valid-python-version() {
```

Comment:
this might still be useful -- I think there's still ways people can manually get out of sync ?